config/karma-docs.conf.js

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
@matthewrfindley - just realized that this is config/**karma-docs**.conf.js. Pleease disregard what I was saying...you already did it. LOL.